WcaOnRails/app/webpacker/lib/helpers/user-groups-and-roles-constants.js

```diff
@@ -0,0 +1,19 @@
+export const GROUP_TYPE = {
+  DELEGATE_PROBATION: 'delegate_probation',
+  DELEGATE_REGIONS: 'delegate_regions',
+  TEAMS_COMMITTEES: 'teams_committees',
+};
+
+export const DELEGATE_STATUS = {
+  SENIOR_DELEGATE: 'senior_delegate',
+  REGIONAL_DELEGATE: 'regional_delegate',
+  DELEGATE: 'delegate',
+  JUNIOR_DELEGATE: 'candidate_delegate',
+  TRAINEE_DELEGATE: 'trainee_delegate',
+};
+
+export const TEAMS_COMMITTEES_STATUS = {
+  MEMBER: 'member',
+  SENIOR_MEMBER: 'senior_member',
+  LEADER: 'leader',
+};
```

Comment:
See my remark about exporting these enums from Rails via `wca-data.js.erb` in another PR.

Reply:
Done